.gitattributes

```diff
@@ -9,3 +9,4 @@ drivers/* linguist-vendored
 *.py eol=lf
 *.hpp eol=lf
 *.xml eol=lf
+*.natvis eol=lf
```

Comment:
There's a missing LF at EOF here (not critical, but as seen above it's reported as non-normal).